code/parachain/runtime/primitives/src/currency.rs

```diff
@@ -89,9 +89,24 @@ macro_rules! list_assets {
 			}
 		}
 
-		pub fn list_assets() -> Vec<Asset> {
+		const RELAY_NATIVE_CURRENCIES: [CurrencyId; 1] = [CurrencyId::KSM];
```

Comment:
Does this (and known_location) need to be in the macro?

Reply:
Not exactly, it made sense for me as it was only used there but it can be extracted if it makes more sense

Reply:
I will resolve this to get it merged asap and can extract it later on some other PR